spacy/lang/fr/stop_words.py

```diff
@@ -36,45 +37,48 @@
 l' l’ la laisser laquelle le lequel les lesquelles lesquels leur leurs longtemps
 lors lorsque lui lui-meme lui-même là lès
 
-m' m’ ma maint maintenant mais malgre me meme memes merci mes mien
+m' m’ ma maint maintenant mais malgre malgré me meme memes merci mes mien
 mienne miennes miens mille moi moi-meme moi-même moindres moins
 mon même mêmes
 
 n' n’ na ne neanmoins neuvième ni nombreuses nombreux nos notamment
-notre nous nous-mêmes nouvea nul néanmoins nôtre nôtres
+notre nous nous-mêmes nouveau nul néanmoins nôtre nôtres
 
-o ô on ont onze onzième ore ou ouias oust outre
+o ô on ont onze onzième or ou ouias ouste outre
 ouvert ouverte ouverts où
 
-par parce parfois parle parlent parler parmi parseme partant
+par parce parfois parle parlent parler parmi partant
 pas pendant pense permet personne peu peut peuvent peux plus
-plusieurs plutôt possible possibles pour pourquoi
-pourrais pourrait pouvait prealable precisement premier première premièrement
-pres procedant proche près pu puis puisque
+plusieurs plutot plutôt possible possibles pour pourquoi
+pourrais pourrait pouvait prealable precisement
+premier première premièrement
+pres procedant proche près préalable précisement pu puis puisque
```

Comment:
- removed "parseme" (conjugated form of verb parsemer, carries meaning);
- added "plutot" (unnaccented);
- added "préalable" (correct form, with accent);
- added "précisement" (correct form, with accent);